x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/security/authz/store/ReservedRolesStore.java

```diff
@@ -678,9 +678,17 @@ public static RoleDescriptor kibanaSystemRoleDescriptor(String name) {
                 RoleDescriptor.IndicesPrivileges.builder().indices("metrics-endpoint.policy-*").privileges("read").build(),
                 // Endpoint metrics. Kibana requires read access to send telemetry
                 RoleDescriptor.IndicesPrivileges.builder().indices("metrics-endpoint.metrics-*").privileges("read").build(),
-                // Fleet package upgrade
+                // Fleet package install and upgrade
                 RoleDescriptor.IndicesPrivileges.builder()
-                    .indices("logs-*", "synthetics-*", "traces-*", "/metrics-.*&~(metrics-endpoint\\.metadata_current_default)/")
+                    .indices(
+                        "logs-*",
+                        "synthetics-*",
+                        "traces-*",
+                        "/metrics-.*&~(metrics-endpoint\\.metadata_current_default)/",
```

Comment:
@joshdover Seeing this I wonder if we should be more specific. Instead of having `metrics-*` we should use `metrics-*-*` so it would also not match the other index by "accident". @jpountz Any thoughts?

BTW same applies for the other indices.

Reply:
> Instead of having `metrics-*` we should use `metrics-*-*` so it would also not match the other index by "accident".

Seems like a good idea to me. This would better match the global index templates installed by Elasticsearch as well which match `logs-*-*` and `metrics-*-*` 👍 

I'll add this to #80332